packages/remix-server-runtime/routes.ts

```diff
@@ -87,7 +87,9 @@ export function createStaticHandlerDataRoutes(
       id: route.id,
       path: route.path,
       loader: route.module.loader
-        ? (args: LoaderFunctionArgs) =>
+        ? // Need to use RR's version here to permit the optional context even
+          // though we know it'll always be provided in remix
+          (args: RRLoaderFunctionArgs) =>
```

Comment:
At some point it could make sense to change RR from `context?:  unknown` to `context: unknown | undefined` which would, I think, eliminate this typing oddity.